README.md

```diff
@@ -33,12 +33,13 @@ public function registerBundles()
 ```
 3. Create tables for changes storing
 ```
-app/console doctrine:schema:update --force
+bin/console doctrine:schema:update --force
 ```
 4. Configure mapping if needed.
 
 Usage
```

Comment:
i think is better to use `# header` notation, because it's looks much cleaner and allows to explicitly define header level (one sharp for first, two for second, etc ...)

Reply:
I think it will be too bold while reading text